knowledge-base/kb-window-howto-display-windowsformshost-in-radwindow.md

```diff
@@ -0,0 +1,47 @@
+---
+title: How to Display WindowsFormsHost in RadWindow
```

Comment:
It is recommended for the 'title', 'description' and 'page_title' fields to contain different ways of describing the customer context. In other words make sure that there are not duplicated fields. For example, if the page_title is "How to Display WindowsFormsHost in RadWindow". The title could be "Showing WinForms host in RadWindow". 

Also, avoid using unnecessary words in those files. For example, like "This article explains..". Keep it simple and using the customer context.

Read more about those suggestions in the [Knowledge Base Content Standard](https://wiki.progress.com/pages/viewpage.action?spaceKey=DTS&title=Knowledge+Base+Content+Standards#KnowledgeBaseContentStandards-Description) wiki article.